tests/framework/testutils/helpters.go

```diff
@@ -35,3 +38,28 @@ func KeyValuesFromGetResponse(resp *clientv3.GetResponse) (kvs []KV) {
 	}
 	return kvs
 }
+
+func KeyValuesFromWatchResponse(resp clientv3.WatchResponse) (kvs []KV) {
+	for _, event := range resp.Events {
+		kvs = append(kvs, KV{Key: string(event.Kv.Key), Val: string(event.Kv.Value)})
+	}
+	return kvs
+}
+
+func KeyValuesFromWatchChan(wch clientv3.WatchChan, wantedLen int, timeout time.Duration) (kvs []KV, err error) {
+ForLoop:
```

Comment:
No need for this label if we use return instead.

Reply:
removed.